roles/create-single-k8s-cluster-with-minikube/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+- name: Create a single k8s master cluster by kubeadm
+  shell: |
+    set -ex
+    # Download minikube binary, the newest version supportted is v0.30.0
```

Comment:
s/the newest version supportted is v0.30.0/the supported latest version is v0.30.0/

Reply:
done